Stylesheets/components/utility.css

```diff
@@ -0,0 +1,36 @@
+@import url(/Stylesheets/main.css);
+
+.heading-text,
+.para-text {
+  padding: 10px;
+}
+.body-cp-lg {
+  font-size: 2rem;
+  line-height: 2.5rem;
+}
+.body-cp-rg {
+  font-size: 1.5rem;
+  line-height: 2.5rem;
+}
+.body-cp-md {
+  font-size: 1rem;
+  line-height: 2.5rem;
+}
+.body-cp-sm {
+  font-size: 0.75rem;
+  line-height: 2.5rem;
```

Comment:
If the line-height is same for all the properties, move it to a common property and use it across.

Reply:
oh ya i changed it ty :)